src/pages/HomePage.jsx

```
@@ -0,0 +1,21 @@
+import Hero from "../components/Hero.jsx";
+import CTA from "../components/CTA.jsx";
+import OurWork from "../components/OurWork.jsx";
+import AboutUs from "../components/AboutUs.jsx";
+import Services from "../components/Services.jsx";
+import KeyHighlights from "../components/KeyHighlights.jsx";
```

ESLint

[error] 'KeyHighlights' is defined but never used.

(no-unused-vars)